resources/public/api/conf/1.0/employment_expenses_retrieve_openApiFeatureTest.yaml

```diff
@@ -0,0 +1,110 @@
+summary: Retrieve Employment Expenses
```

Comment:
For the purposes of the integration tests you might be able to place the 'switched' test file away from the production resources in test/resources/public/api/conf/1.0

Reply:
That could make it tidier, & just include the switch config in the unit tests and ITs.

Wondering if it would be preferable to leave it as-is though, as it currently shows exactly how to make it work; also easier to demo or try out?